pkg/build/archive.go

```diff
@@ -80,6 +81,15 @@ func Archive(manifest model.Manifest) error {
 		if err := sanitizeTemplate(manifest, path.Join(out, "install/kubernetes/operator/profiles/default.yaml")); err != nil {
 			return fmt.Errorf("failed to sanitize charts")
 		}
+		if err := util.CopyDir(path.Join(manifest.RepoDir("operator"), "deploy"), path.Join(out, "install/kubernetes/operator/deploy")); err != nil {
+			return err
+		}
+		if err := sanitizeTemplate(manifest, path.Join(out, "install/kubernetes/operator/deploy/operator.yaml")); err != nil {
+			return fmt.Errorf("failed to sanitize operator manifest")
+		}
+		if err := util.CopyDir(path.Join(manifest.RepoDir("operator"), "samples"), path.Join(out, "samples/operator")); err != nil {
+			return err
+		}
```

Comment:
Does it make sense to sanitize the samples folder as well? It currently doesn't have any hub or tag but it could be added in the future, no? Then we would be safe.